requirements.txt

```diff
@@ -7,5 +7,5 @@ nlohmann/json@v3.11.2 -DJSON_MultipleHeaders=ON -DJSON_BuildTests=Off
 ROCm/FunctionalPlus@v0.2.18-p0
 ROCm/eigen@3.4.0
 ROCm/frugally-deep@9683d557eb672ee2304f80f6682c51242d748a50
-ROCm/composable_kernel@4b798833c4cc614bbf7f7fb43c5ee81d12602f05 -DCMAKE_BUILD_TYPE=Release
+ROCm/composable_kernel@0c9012fb70bcd2750ff0d5b8c23e4bc6f5937709 -DCMAKE_BUILD_TYPE=Release
```

Comment:
We should coordinate this with #3379.
I think Jun's update should be new enough to have the changes you need.

Reply:
Yes Jun's update is a newer hash, will drop this hash change waiting for that PR to finish first.

Reply:
commit hash is set to latest version from develop